documentation/assemblies/kafka-connect/assembly-kafka-connect.adoc

```diff
@@ -0,0 +1,34 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-kafka-connect-{context}']
+= Kafka Connect cluster configuration
```

Comment:
Since this now covers also Connect S2I, should we name this `Kafka Connect and Connect S2I cluster configuration` or something like that? We had separate chapters before, so it might be confusing if just one disappears now.

Reply:
To avoid it getting too long, I though this: 

> Kafka Connect cluster configuration (plus S2I option)

Reply:
Yeah, I think that works ... or maybe _(including S2I)_ could be even better?

Reply:
Also worth updating the first sentence to refer to S2I.

Reply:
Changed my mind about this. Thought this looked neater (and we use it elsewhere in guide)

> Kafka Connect/S2I cluster configuration

Reply:
![image](https://user-images.githubusercontent.com/47596553/95842416-6ba45f80-0d3e-11eb-8f3a-dee0406d9438.png)

Reply:
Looks fine to me.